app.js

```diff
@@ -5,7 +5,7 @@ const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 
 // setting up routes
-const index = require('./routes/index');
+// const index = require('./routes/index');
```

Comment:
Yeah I'd like to rewrite that routing later, it's pretty gross. For now it's what worked with react-router